cylc/flow/scripts/lint.py

```diff
@@ -38,7 +38,15 @@
 from optparse import Values
 from pathlib import Path
 import re
-import tomli
+try:
+    # BACK COMPAT: tomli
+    #   Support for Python versions before tomllib was added to the
+    #   standard library.
+    # FROM: Python 3.7
+    # TO: Python: 3.10
+    from tomli import loads as toml_loads, TOMLDecodeError
+except ImportError:
+    from tomllib import loads as toml_loads, TOMLDecodeError
```

Comment:
The tomli library changed its namespace when entering the standard library (same as importlib_metadata) so needed this try/except.